src/scripts/destinyTrackerApi/itemTransformer.js

```diff
@@ -0,0 +1,35 @@
+class itemTransformer {
```

Comment:
IMO this doesn't really deserve to be an injectable, instantiable object - it's just a bunch of functions.